var/spack/repos/builtin/packages/morphio/package.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2013-2018 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Morphio(CMakePackage):
+    """Library for reading / writing morphology files"""
+
+    homepage = "https://github.com/BlueBrain/MorphIO"
+    url      = "git@github.com:BlueBrain/MorphIO.git"
```

Comment:
url with https ?

Reply:
Same as for `py-morphio`.

Reply:
I didnt get you....MorphIO is public now..so we should change both then